retriever/engines/msaccess.py

```diff
@@ -1,8 +1,8 @@
-from __future__ import print_function
```

Comment:
Python 3, thought I think there's a typo and @henrykironde meant Python 2 in this case. Without the import lines that use print like a function `print(...)` will error in Python 2.